generators/entity/prompts.js

```diff
@@ -242,10 +242,11 @@ function askForTableName() {
     // don't prompt if there are no relationships
     var entityTableName = this.entityTableName;
     var prodDatabaseType = this.prodDatabaseType;
-    if (this.relationships.length === 0 || !((prodDatabaseType === 'oracle' && entityTableName.length > 14) || entityTableName.length > 30)) {
+    console.log(entityTableName);
```

Comment:
Shit ya. Forgot to remove it

Thanks & regards,
Deepu

On 5 Sep 2016 17:44, "Sendil Kumar N" <notifications@github.com> wrote:

> In generators/entity/prompts.js
> <https://github.com/jhipster/generator-jhipster/pull/4062#discussion_r77513129>
> :
>
> > @@ -242,10 +242,11 @@ function askForTableName() {
> >      // don't prompt if there are no relationships
> >      var entityTableName = this.entityTableName;
> >      var prodDatabaseType = this.prodDatabaseType;
> > -    if (this.relationships.length === 0 || !((prodDatabaseType === 'oracle' && entityTableName.length > 14) || entityTableName.length > 30)) {
> > +    console.log(entityTableName);
>
> Testing ?? console.log?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/4062/files/af28cd8aa53159fe21db25d47151bd5cfaf3faec#r77513129>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABDlF3DQxdjpCs3TipIAjdAt2FQ2bPfpks5qnAengaJpZM4J0Wqj>
> .
>